### DISCUSSION TITLE
Injecting JS with Stylus?

### DISCUSSION THREAD

**Discussion by 0D1nn8502** [openstyles/stylus#1817] [Q&A] [answered]
We are two university students working on a capstone project on making Indian government sites more user-friendly through a Chrome extension. We can use stylus to modify css but wanted to know if there is a way to inject Javascript as well? Can we talk to someone ?

**Comment by pabli24** [comment, marked as answer]
Stylus is CSS only.
Use [Tampermonkey](https://chromewebstore.google.com/detail/tampermonkey/dhdgffkkebhmkfjojejmpbldmpobfkfo) or [Violentmonkey](https://chromewebstore.google.com/detail/violentmonkey/jinjaccalgkegednnccohejagnlnfdag)
